README.md

```diff
@@ -19,24 +19,24 @@ Another Android Monero Wallet for Monero
 Help us translate Monerujo! You can find instructions for adding a new translation or updating an existent one in [this guide](https://github.com/monero-ecosystem/monero-translations/blob/master/translate-monerujo.md), but is suggested to contact the Monero Localization Workgroup first if you have any doubt or question. You can do so in many ways. For example by email: translate@getmonero.org or chatting in `#monero-translations` (chatroom on Freenode, matrix and MatterMost). To see the complete list of contacts, take a look at the [official repository of the workgroup on GitHub](https://github.com/monero-ecosystem/monero-translations/blob/master/README.md#contacts).
 
 ### Disclaimer
-You may lose all your Moneroj if you use this App. Be cautious when spending on the mainnet.
```

Comment:
Moneroj is the plural of Monero, but I see how people reading the README would thtink it's a typo

Reply:
Oh, i didn't knew that, sorry